doc/content/integrations/webhooks/cloud-integrations/tagoio/tagoio-setup/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "TagoIO Setup"
+title: "Tago Setup"
```

Comment:
I still see their name as `TagoIO` on their website. Is this change intentional ?